### PR TITLE
release-checklist: minor updates

### PR DESCRIPTION
**Pull request by bgilbert**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-checklist.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ Release checklist:
      - [ ] If doing a branched release, also include a PR to merge the NEWS changes into master
  - [ ] Ensure your local copy is up to date with master and your working directory is clean
  - [ ] Ensure you can sign commits and any yubikeys/smartcards are plugged in
- - [ ] Run `./tag_release <vX.Y.z> <git commit hash>`
+ - [ ] Run `./tag_release.sh <vX.Y.z> <git commit hash>`
  - [ ] Push that tag to Github
  - [ ] Run `./build_releases`
  - [ ] Sign the release artifacts by running
```
Comment (ajeddeloh) [inline .github/ISSUE_TEMPLATE/release-checklist.md]: Can you add a ! on the subkey id? Not part of this change but I keep forgetting to update it.

Comment (bgilbert) [reply .github/ISSUE_TEMPLATE/release-checklist.md]: Done.
